ide/projectui/src/org/netbeans/modules/project/ui/ProjectUtilities.java

```diff
@@ -135,28 +130,25 @@ private void doClose(Project[] projects, boolean notifyUI, Wrapper wr) {
                 if (dobj != null) {
                     FileObject fobj = dobj.getPrimaryFile();
                     Project owner = ProjectConvertors.getNonConvertorOwner(fobj);
-                    ERR.log(Level.FINER, "Found {0} owned by {1} in {2} of {3}", new Object[] {fobj, owner, tc.getName(), tc.getClass()});
+                    LOG.log(Level.FINER, "Found {0} owned by {1} in {2} of {3}", new Object[] {fobj, owner, tc.getName(), tc.getClass()});
 
-                    if (listOfProjects.contains(owner)) {
```

Comment:
It would be cleaner to just get `Set<String> files = project2FilesMap.get(owner);` here and change the test to `files != null`. Then change the earlier loop to just `for (Project p : projects)` and get rid of listOfProjects altogether.

Reply:
good idea - will give it a try!

Reply:
done